src/Tools/ExternalAccess/RazorCompiler/Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler.csproj

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
+
+    <!-- NuGet -->
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler</PackageId>
+    <PackageDescription>
+      A supporting package for Razor source generator:
+      https://github.com/dotnet/razor
+    </PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!--
+      ⚠ ONLY RAZOR ASSEMBLIES MAY BE ADDED HERE ⚠
+    -->
+    <InternalsVisibleTo Include="Microsoft.NET.Sdk.Razor.SourceGenerators" Key="$(RazorKey)" />
```

Comment:
Are we sure that we don't need any of the other Razor assemblies here?

Reply:
Let's also confirm that this assembly is signed with the same key used by the IDE assemblies, it's very possible that it's not since we combined repos.

Reply:
> Are we sure that we don't need any of the other Razor assemblies here?

I'm not sure, maybe @chsienki can answer? But I guess more can be added later easily (when the Razor side is implemented)?

> Let's also confirm that this assembly is signed with the same key used by the IDE assemblies

No idea if this is the correct way to verify that, but

```ps1
sn -Tp Microsoft.NET.Sdk.Razor.SourceGenerators.dll
```

shows the same key which is set as `$(RazorKey)` in `eng/targets/Settings.props`.